internal/controller/clusterinstance_controller.go

```diff
@@ -53,6 +53,8 @@ type ClusterInstanceReconciler struct {
 	TmplEngine *ci.TemplateEngine
 }
 
+type MapRenderedObjects map[int][]ci.RenderedObject
```

Comment:
This is not a particularly useful type name.

Maybe `SyncWaveMap` would be better or `SyncWaveObjects` or something?

This current name just doesn't make the use or the structure clear.

Reply:
I like `SyncWaveMap` - updated to this!